modules/how-service-accounts-assume-aws-iam-roles-in-default-projects.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * authentication/assuming-an-aws-iam-role-for-a-service-account.adoc
+// * rosa_architecture/rosa_policy_service_definition/rosa-sre-access.adoc
+
+:_content-type: CONCEPT
+[id="how-service-accounts-assume-aws-iam-roles-in-default-projects_{context}"]
+= How service accounts assume AWS IAM roles in default projects
+
+When you install a {product-title} cluster that uses the AWS Security Token Service (STS), cluster-specific Operator AWS Identity and Access Management (IAM) roles are created. These IAM roles permit the ROSA cluster Operators to carry out core OpenShift functionality.
+
+Cluster Operators use service accounts to assume IAM roles. When a service account assumes an IAM role, temporary STS credentials are provided for the service account to use in the cluster Operator's pod. If the assumed role has the necessary AWS privileges, the service account can run AWS SDK operations in the pod.
+
+[discrete]
+[id="workflow-for-assuming-aws-iam-roles-in-default-projects_{context}"]
+== Workflow for assuming AWS IAM roles in default projects
+
+// The following diagram illustrates the workflow by which the service accounts for cluster Operators assume IAM roles:
+//
+// .Workflow for assuming AWS IAM roles in default projects 
+// TODO: Add diagram.
+
+The workflow has the following stages:
+
+. Within each project that a cluster Operator runs, a deployment spec defines a service account token configuration. These tokens are audience- and time-bound. Every hour, ROSA generates a new token, which generates the following temporary credentials mounted to the cluster Operator's pod:
+
+** An `$AWS_ARN_ROLE` environment variable that contains the ARN for the IAM role that has the permissions required to run AWS SDK operations.
+
+** An `$AWS_WEB_IDENTITY_TOKEN_FILE` environment variable that contains the full path in the pod to the OpenID Connect (OIDC) token for the service account. The full path is `/var/run/secrets/eks.amazonaws.com/serviceaccount/token`.
```

Comment:
```suggestion
** An `$AWS_WEB_IDENTITY_TOKEN_FILE` variable that contains the full path in the pod to the OpenID Connect (OIDC) token for the service account. The full path is `/var/run/secrets/openshift/serviceaccount/token`.
```

Reply:
Is this for OpenShift cluster operators or the pod identity webhook? If its for the pod identity webhook then ignore this suggestion as the eks path is correct.

Reply:
This is for the cluster Operators, so I corrected the path as suggested.